test/unit/mdc-textfield/mdc-text-field.test.js

```diff
@@ -103,8 +122,9 @@ function setupTest() {
   const root = getFixture();
   const icon = root.querySelector('.mdc-text-field__icon');
   const bottomLine = new FakeBottomLine();
-  const component = new MDCTextField(root, undefined, (el) => new FakeRipple(el), () => bottomLine);
-  return {root, bottomLine, icon, component};
+  const helperText = new FakeHelperText();
```

Comment:
I don't think you need to modify the setupTest logic. The only reason I added bottomLine was because I referenced it from some tests (like 'adapter.registerBottomLineEventHandler adds event listener to bottom line').

Reply:
I added two tests for checking if the helper text is initialized on the correct element and with the correct factory...should I remove these too?

Reply:
Added a comment above. I think the second test is a good test.